cmd/poller/options/options.go

```diff
@@ -65,6 +65,13 @@ func SetPathsAndHostname(args *Options) {
 	}
 
 	args.HomePath = conf.GetHarvestHomePath()
+	configPath := os.Getenv("HARVEST_CONF")
+	if configPath != "" {
+		if !strings.HasSuffix(configPath, "/") {
+			configPath += "/"
+		}
+		args.HomePath = configPath
+	}
```

Comment:
can we move this logic to conf.go file in GetHarvestHomePath function ?

Reply:
i considered that initially, but was concerned about how frequently that method is called. After profiling it though, it's fine since most of these happen via initialization. I like it, thanks! I'll move this to `GetHarvestHomePath`

Also fixed the `grafana` problem Yann mentioned with this PR

Reply:
As it's moved to conf.go, remove this from here as well.

Reply:
oh dear :) good catch